ios/RNSScreenStackHeaderConfig.m

```diff
@@ -138,10 +138,21 @@ - (void)updateViewControllerIfNeeded
   }
 }
 
+- (void)layoutNavigtationController
```

Comment:
This name doesn't reflect what this method does, also a typo 🙊

Reply:
I added View to the name of the function